front/app/modules/commercial/content_builder/services/ContentBuilder.ts

```diff
@@ -0,0 +1,48 @@
+import { API_PATH } from 'containers/App/constants';
+import streams from 'utils/streams';
+
+export interface IBuilderLayoutData {
+  type: 'project';
```

Comment:
Is the type here project? I expected something like `content_builder_layout`.

Reply:
@IvaKop This is another case where I think the specification confused me a bit. It states "content_buildable: The type and ID of the data instance to which the layout belongs. For this iteration, the type will always be Project." I guess I'm wondering where that "Project" type would appear in the code, if not here?